rust/core-lib/src/client.rs

```diff
@@ -74,6 +74,15 @@ impl Client {
         self.0.send_rpc_notification("available_languages", &json!({ "languages": languages }))
     }
 
+    pub fn content_changed(&self, view_id: ViewId) {
+        self.0.send_rpc_notification(
+            "content_changed",
```

Comment:
Not sure if this has been discussed elsewhere, but I personally feel this this doesn't express that the file _hasn't_ been changed by Xi but by something else. I'm really bad at naming things and the names should be short, but something like `file_externally_changed` would be so much clearer to me as frontend dev

Reply:
I agree, `file_externally_changed` does not seem to be too long.